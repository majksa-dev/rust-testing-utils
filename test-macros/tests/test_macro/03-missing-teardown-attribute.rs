use testing_utils::macros as utils;

fn setup() -> usize {
    42
}

#[utils::test(setup = setup)]
fn single_server(_ctx: &mut usize) {}

fn main() {}
