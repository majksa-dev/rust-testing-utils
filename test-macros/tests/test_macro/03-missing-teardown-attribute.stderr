error: teardown attribute not found
 --> tests/test_macro/03-missing-teardown-attribute.rs:7:1
  |
7 | #[utils::test(setup = setup)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `utils::test` (in Nightly builds, run with -Z macro-backtrace for more info)
